AIMSICD/src/main/java/scratchpad

```diff
@@ -0,0 +1,6 @@
+FEMTOCELL? -> can AIMSICD onPrepareOptionsMenu deleted,
```

Comment:
Then use `git rm --cached AIMSICD/src/main/java/scratchpad` and add it to `.gitignore`